exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/Blockchain.java

```diff
@@ -169,13 +182,38 @@ public long getHeight() {
     return schema.getBlocks();
   }
 
+  /**
+   * Returns the block at the given height.
+   *
+   * @param height the height of the block; must be non-negative and less than or equal to
+   *     the current {@linkplain #getHeight() blockchain height}
+   * @return a block at the height
+   * @throws IllegalArgumentException if the height is not valid
+   */
+  public Block getBlock(long height) {
+    checkHeight(height);
+
+    ListIndex<HashCode> blockHashes = getBlockHashes();
+    HashCode blockHash = blockHashes.get(height);
+    MapIndex<HashCode, Block> blocks = getBlocks();
+    return blocks.get(blockHash);
+  }
+
+  private void checkHeight(long height) {
+    long maxHeight = getHeight();
```

Comment:
Use `blockHeight` instead of `height` and `blockchainHeight` instead of `maxHeight`?